internal/cmd/options.go

```diff
@@ -0,0 +1,18 @@
+package cmd
+
+import (
+	"github.com/spf13/pflag"
+)
+
+// SignerFlagOpts cmd opts for using cmd.GetSigner
+type SignerFlagOpts struct {
+	Key      string
+	KeyFile  string
+	CertFile string
+}
+
+func (opts *SignerFlagOpts) ApplyFlag(fs *pflag.FlagSet) {
```

Comment:
Nit: can you add comments to exported methods though I feel it's self-explained? And `ApplyFlags` probably makes more sense.